axes/helpers.py

```diff
@@ -342,7 +342,13 @@ def get_lockout_response(request, credentials: dict = None) -> HttpResponse:
         )
 
     if request.META.get("HTTP_X_REQUESTED_WITH") == "XMLHttpRequest":
-        return JsonResponse(context, status=status)
+            json_response = JsonResponse(context, status=status)
+            json_response["Access-Control-Allow-Origin"] = settings.AXES_ALLOWED_CORS_ORIGINS
+            json_response["Access-Control-Allow-Methods"] = "POST, OPTIONS"
```

Comment:
These should be parametrisable. Why are POST and OPTIONS allowed specifically?

Reply:
@aleksihakli because those are the ones needed to perform AJAX calls, without them it's not possible they need to be there by default.

Reply:
@aleksihakli i've update the description to state that the problem are ajax calls

Reply:
Would HEAD and possibly GET methods also be good to include or are they obsolete in this specific endpoint?

Reply:
For an AJAX call are not necessary and this is only about those calls
("HTTP_X_REQUESTED_WITH") == "XMLHttpRequest" means that only happens in the case of an AJAX call